### ISSUE TITLE
Netdata dashboard is outdated

### ISSUE THREAD

**Issue by OdysLam** [Homebrew/homebrew-core#72564]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

Not relevant, netdata was installed as expected.

### 

- [X] The `brew doctor` above contains no "Warning" lines.

### What were you trying to do (and why)?

Use Netdata

### What happened (include all command output)?

I installed the Netdata Agent, which works great but
1) I can't connect to Netdata Cloud
2) The dashboard is several versions old (while Netdata Agent was actually the latest -1.28)

### What did you expect to happen?

1) Ability to claim node to Netdata Cloud
2) See the latest version of Netdata Agent dashboard

### Step-by-step reproduction instructions (by running `brew` commands)

Brew install Netdata

**Comment by SMillerDev** [comment]
https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by OdysLam** [comment]
On a relevant note, I am actually the DevRel lead on Netdata, so I would love to engage with you folks to get Netdata working perfectly with brew, which is my go-to choice for installing software on my Mac. 

Regarding the Netdata Cloud issue, we released a couple of months back a new version of Netdata which doesn't require the vendoring of libraries from our side (apart from libmosquitto to which we have applied some patches that do not exist upstream). I remember that this was the main blocker behind not building Netdata with Cloud support.

Finally, on our [Community Forums](https://community.netdata.cloud) we have created a whole category for people who maintain Netdata on various platforms, as a direct channel of communication between the maintainers and Netdata Engineers. If you are interested to join,  tell me to add you to the closed group!
